### ISSUE TITLE
Reach out to current Summit participants

### ISSUE THREAD

**Issue by DinisCruz** [OpenSecuritySummit/oss2018#312]
(this should be an individual email, that contains a link and screenshot/pdf of the current participant page)

And ask
 - review their current participation and sessions that they are organizing
 - ask for other topics we should be addressing in the Summit
 - ask to promote the summit on their network (blog and twiter)
 - asked them to reach out to their network and see if they can find one more participant to the Summit

**Comment by VasBu** [comment]
I just want to share information that on Friday (04/May/2018) I sent email messages to the organizers of sessions for the following summit.

I took the following approach:
- I used working sessions page on the website https://open-security-summit.org/working-sessions/ to go though all declared working sessions and identify the already assigned organizers
- I checked the central sheet file, tab "OSS 17-18 Contacts" ((https://docs.google.com/spreadsheets/d/1Sx15O_pGQr8jJvY4JoUYXI-RaWtKabuZU8rp85wHqTQ/edit#gid=698941056)) to find the contact information. If email address was missing I was searching for and I updated the Google Spreadsheet.
- I used template message written by @almeidam to send message to each and every organizer (CC @DinisCruz )
- After I sent a "call for action" message for review the description for sessions, I updated the Spreadsheet that the message has been sent.

**Comment by VasBu** [comment]
Today, 07-may-2018, I went through the current list of the user sessions and sent email to the organizers that has left.

I took the following approach:

- I used user sessions page on the website https://open-security-summit.org/user-sessions/ to go though all declared working sessions and identify the already assigned organizers
- I checked the central sheet file, tab "OSS 17-18 Contacts" ((https://docs.google.com/spreadsheets/d/1Sx15O_pGQr8jJvY4JoUYXI-RaWtKabuZU8rp85wHqTQ/edit#gid=698941056)) to find the contact information. If email address was missing I was searching for and I updated the Google Spreadsheet.
- I used template message written by @almeidam to send message to each and every organizer (CC @DinisCruz )
- After I sent a "call for action" message for review the description for sessions, I updated the Spreadsheet that the message has been sent.
